CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## [Unreleased]
 
+## [0.4.2] - 2024-04-16
+
+### Behavior Changes
+
+* Periodically flush unsynced bytes when rewriting to avoid I/O jitters if flushing too many bytes impede the foreground writes. #347
```

Comment:
To be consistent with github release notes:
```suggestion
* Periodically flush unsynced bytes when rewriting to avoid I/O jitters if flushing too many bytes impede the foreground writes. (#347)
```

Usually the note should be added right in the original PR, so that git blame is enough to track history.